README.md

```diff
@@ -307,8 +322,17 @@ const streamCamera = new StreamCamera({
 - `exposureCompensation: number` - _Range: `-10`-`10`; Default: `0`_
 - [`exposureMode: ExposureMode`](#exposuremode) - _Default: `ExposureMode.Auto`_
 - [`awbMode: AwbMode`](#awbmode) - _Default: `AwbMode.Auto`_
+- `awbGains: [number, number]` - _Default: `null`_
 - `analogGain: number` - _Default: `0`_
 - `digitalGain: number` - _Default: `0`_
+- `colourEffect: [number, number]` - _Default: `[0,0]`_
+- [`imageEffect: ImxfxMode`](#imageeffect) - _Default: `ImxfxMode.None`_
+- [`dynamicRange: DynamicRange`](#dynamicrange) - _Default: `DynamicRange.Off`_
+- `videoStabilisation: boolean` - _Default: `false`_
+- [`meteringMode`](#meteringMode) - _Default: `MeteringMode.Off`_
+- [`flickerMode`](#flickerMode) - _Default: `null`_
```

Comment:
Is this off by default? Could we mention that here?

Reply:
It's off by default. Yes